docs/tutorials/polybeam_perturb_simulation.ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
<p>suggest having all imports at the top</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>